oracle/datadog_checks/oracle/queries.py

```diff
@@ -5,7 +5,7 @@
 
 ProcessMetrics = {
     'name': 'process',
-    'query': 'SELECT PROGRAM, PGA_USED_MEM, PGA_ALLOC_MEM, PGA_FREEABLE_MEM, PGA_MAX_MEM FROM GV$PROCESS',
+    'query': 'SELECT PROGRAM, SUM(PGA_USED_MEM), SUM(PGA_ALLOC_MEM), SUM(PGA_FREEABLE_MEM), SUM(PGA_MAX_MEM) FROM GV$PROCESS GROUP BY PROGRAM',
```

Comment:
What's the purpose of summing up `pga_max_mem`? `pga_max_mem` is a high usage watermark for every process. But since every process generally reached its max usage at a different time, their sum will be greater than the total memory ever allocated.

Reply:
That's a good point; the sum of all processes' maximums isn't the most useful. 

For some context, the main goal for our company is to have a good idea of the PGA state (used, total, etc.). Our current dashboards are, for example, summing up oracle.process.pga_used_memory and reporting that as amount of PGA used. Based on my research that is incorrect because only one value is reported per program when multiple exist:

select program, count(*) from GV$PROCESS group by program order by 2 desc;
oracle@cs12dbnrduva01	356
oracle@cs12dbnrduva01 (SCMN)	2

My first thought was to sum them all, grouped by program so that there is no information missing. Do you have any thoughts on how else we can bring in all that missing information?

Reply:
Maybe we can add PID column to query so that the entries with duplicate program names are not overridden. For example:
SELECT PROGRAM, PGA_USED_MEM, PGA_ALLOC_MEM, PGA_FREEABLE_MEM, PGA_MAX_MEM FROM GV$PROCESS

Reply:
Yes, `program` can't be the key because generally a single `program` can have multiple sessions/processes. 

Summing memory allocations and grouping by `program` solves this - except for `pga_max_mem` where summing doesn't produce a meaningful value.

An alternative solution, adding `PID` as a unique identifier seems correct to me. The aggregations  would need to be performed outside of database in that case.

Reply:
My latest commit includes the original query with the pid column added. `ddev test oracle` passes on my machine. 

Previously existing monitors seem to be functioning with correct values
![dashboard-pid](https://user-images.githubusercontent.com/127704798/225079911-db20dfdd-93f3-41b1-b513-83c8d439bc82.PNG)

And all metrics appear to be reporting properly
![metrics-with-pid](https://user-images.githubusercontent.com/127704798/225079804-a1b3740a-7f85-4c0e-a806-94bba5766d4e.PNG)

Reply:
@nenadnoveljic Is there anything else that should be done before this can be merged?